docs/source/config.rst

```diff
@@ -5,12 +5,181 @@ Everest configuration
 *********************
 
 Everest is configured via a yaml file, using a set of pre-defined keywords that
-are described in more detail in the section :ref:`cha_config_reference`.
+are described in more detail in the :ref:`keyword reference <cha_config_reference>`.
+
+The configuration file has several sections which defined and explained below. The sections can appear in any arbitrary order. The order of the sections presented below is a logical sequence of the various sources of information necessary to run an optimization experiment.
+
+The configuration snippets below are only an example of the different elements necessary in a section. Every case will have different information in a section depending on the problem being optimized.
+
+The egg model is used as an example in the next sections. You can download it :download:`here <../../examples/egg/everest/model/config_flow.yml>`.
```

Comment:
I'm _assuming_ so. The `download` directive creates a file in the build directory. What I have in my local build is a file with `everest/docs/build/html/_downloads/[hash]/config_flow.yml`. So I'm hoping that this would work in readthedocs as well!